l1-contracts/scripts/register-hyperchain.ts

```diff
@@ -106,7 +107,15 @@ async function main() {
         await deployer.registerToken(baseTokenAddress, useGovernance);
       }
 
+      const tokenMultiplierSetterAddress =
+        cmd.tokenMultiplierSetterAddress ||
+        Wallet.fromMnemonic(process.env.MNEMONIC ? process.env.MNEMONIC : ethTestConfig.mnemonic, "m/44'/60'/0'/0/2")
```

Comment:
The default option should be zero address instead

Reply:
I.e. `0x0000000000000000000000000000000000000000` instead of `Wallet.fromMnemonic(process.env.MNEMONIC ? process.env.MNEMONIC : ethTestConfig.mnemonic, "m/44'/60'/0'/0/2")`?

Reply:
fixed as discussed